packages/shared/components/modals/InitProfileActionsModal.svelte

```diff
@@ -17,6 +22,36 @@
         })
         modal?.close()
     }
+
+    function openUnlockStrongholdPopup(): void {
+        openPopup({
+            id: PopupId.UnlockStronghold,
+            props: {
+                onSuccess: () => {
+                    openPopup({
+                        id: PopupId.GetSeedPopup,
+                    })
+                },
+                onCancelled: () => {},
+            },
+        })
+    }
+
+    async function backupSeed(): Promise<void> {
+        if (!$profileManager) {
```

Comment:
we might need to destroy the manager because atm it crashes if you run the seed back tool and then try to login